### PR TITLE
OSIDB-3270: Refactor log messages

### PR DESCRIPTION
**Pull request by MrMarble**
# OSIDB-3270: Refactor log messages

## Checklist:

- [x] Commits consolidated
-  ~Changelog updated~
-  ~Test cases added/updated~
- [x] Jira ticket updated

## Summary:

Now that we have access to the logs on production it makes sense to provide some extra information on where the log was originated, so I used the format `fileName::functionName() [message]` for all the logs we have currently, this is not enforced, so we are free to integrate with other formats (or none) if needed.

I also changed some of the log messages to `debug` so they are excluded in production

Closes OSIDB-3270

### CODE REVIEW

File: src/stores/UserStore.ts [hunk 1]
```diff
@@ -204,48 +204,50 @@ export const useUserStore = defineStore('UserStore', () => {
...
-          console.log('UserStore unauthenticated path not slash to login');
+          console.log('UserStore::isAuthenticated() UserStore unauthenticated path not slash to login');
           router.push({ name: 'login', query });
           return;
         }
-        console.log('UserStore unauthenticated to login');
+        console.log('UserStore::isAuthenticated() UserStore unauthenticated to login');
         router.push({
           name: 'login',
         });
```
Comment (MrMarble) [inline src/stores/UserStore.ts:253]: I kept these logs as `log` but I'm not sure if they can be changed to `debug`?

Comment (superbuggy) [reply src/stores/UserStore.ts:253]: Maybe we could remove these as well?

File: src/components/CveRequestForm.vue [hunk 2]
```diff
@@ -59,7 +59,7 @@ function addPublicCveRequestComment() {
   savingComment.value = true;
   // extract uuid from osimLink
   const flawUuidMatch = /\/flaws?\/([^#?]+)/.exec(props.osimLink);
-  console.log('saving comment');
+  console.debug('saving comment');
```
Comment (C-Valen) [inline src/components/CveRequestForm.vue:62]: I think this could be removed

File: src/services/OsidbAuthService.ts [hunk 3]
```diff
@@ -150,7 +150,7 @@ export async function getNextAccessToken() {
     const parsedBody = RefreshResponse.parse(responseBody);
     return parsedBody.access;
   } catch (e) {
-    console.log('error', e);
+    console.log('OsidbAuthService::getNextAccessToken() Error getting access token', e);
```
Comment (superbuggy) [inline src/services/OsidbAuthService.ts:153]: should this be a `console.error`?

File: src/router/index.ts [hunk 4]
```diff
@@ -194,7 +194,7 @@ router.beforeEach(async (to, from) => {
       query.redirect = to.fullPath;
     }
 
-    console.log('not authenticated, redirecting to login page');
+    console.log('router::beforeEach() not authenticated, redirecting to login page');
```
Comment (superbuggy) [inline src/router/index.ts:197]: Maybe could remove

File: src/router/index.ts [hunk 5]
```diff
@@ -183,7 +183,7 @@ router.beforeEach(async (to, from) => {
   if (!canVisitWithoutAuth && !isAuthenticated) {
     if (isNavigationBackButton) {
       console.log(
-        'unauthenticated (probably just logged out), pressed back, and going to protected route'
+        'router::beforeEach() unauthenticated (probably just logged out), pressed back, and going to protected route'
```
Comment (superbuggy) [inline src/router/index.ts:186]: Maybe could remove

File: src/components/ToastContainer.vue [hunk 6]
```diff
@@ -27,7 +27,7 @@ const height = computed<string>(() => {
 });
 
 function clearAll() {
-  console.log('clearing all');
+  console.debug('clearing all');
```
Comment (C-Valen) [inline src/components/ToastContainer.vue:30]: Candidate to remove?

File: src/composables/useJiraContributors.ts [hunk 7]
```diff
@@ -10,7 +10,7 @@ export default function useJiraContributors(task_key: string) {
   const isLoadingContributors = ref(false);
 
   const { ignoreUpdates } = watchIgnorable(contributors, () => {
-    console.log('Contributors updated');
+    console.debug('Contributors updated');
```
Comment (superbuggy) [inline src/composables/useJiraContributors.ts:13]: @C-Valen Do you think this is still helpful?

File: src/components/widgets/EditableText.vue [hunk 8]
```diff
@@ -40,12 +40,12 @@ function beginEdit() {
 function commit() {
   editing.value = false;
   emit('update:modelValue', editedModelValue.value);
-  console.log('commit');
+  console.debug('commit');
```
Comment (C-Valen) [inline src/components/widgets/EditableText.vue:43]: Candidate to remove?

File: src/components/widgets/EditableText.vue [hunk 9]
```diff
@@ -40,12 +40,12 @@ function beginEdit() {
...
 }
 function abort() {
   editing.value = false;
   editedModelValue.value = props.modelValue;
-  console.log('abort');
+  console.debug('abort');
```
Comment (C-Valen) [inline src/components/widgets/EditableText.vue:48]: Candidate to remove?

File: src/composables/useFlawCommentsModel.ts [hunk 10]
```diff
@@ -43,9 +43,9 @@ export function useFlawCommentsModel(flaw: Ref<ZodFlawType>, isSaving: Ref<boole
       })
       .catch((res) => {
         if (res?.response?.status === 404) {
-          console.error('Error loading internal Jira comments: Task id not found.');
+          console.error('useFlawCommentsModel::loadInternalCommens() Error loading Jira comments: Task id not found.');
```
Comment (superbuggy) [inline src/composables/useFlawCommentsModel.ts:46]:
```suggestion
          console.error('useFlawCommentsModel::loadInternalComments() Error loading Jira comments: Task id not found.');
```

File: src/composables/useJiraContributors.ts (hunk 7 above)
Comment (C-Valen) [inline src/composables/useJiraContributors.ts:13]: Candidate to remove?

File: src/composables/useFlawCommentsModel.ts [hunk 11]
```diff
@@ -43,9 +43,9 @@ export function useFlawCommentsModel(flaw: Ref<ZodFlawType>, isSaving: Ref<boole
...
         } else {
-          console.error('Error loading internal Jira comments');
+          console.error('useFlawCommentsModel::loadInternalCommens() Error loading Jira comments');
```
Comment (superbuggy) [inline src/composables/useFlawCommentsModel.ts:48]:
```suggestion
          console.error('useFlawCommentsModel::loadInternalComments() Error loading Jira comments');
```

File: src/components/widgets/EditableText.vue (hunk 8 above)
Comment (superbuggy) [inline src/components/widgets/EditableText.vue:43]: Could remove

File: src/components/widgets/EditableText.vue (hunk 9 above)
Comment (superbuggy) [inline src/components/widgets/EditableText.vue:48]: Could remove

File: src/components/ToastContainer.vue (hunk 6 above)
Comment (superbuggy) [inline src/components/ToastContainer.vue:30]: Could remove

File: src/components/CveRequestForm.vue (hunk 2 above)
Comment (superbuggy) [inline src/components/CveRequestForm.vue:62]: I think we can remove this

**Review by tboyetteredhat — APPROVED** [review]

File: src/composables/useFlawAffectsModel.ts [hunk 12]
```diff
@@ -23,7 +23,7 @@ export function useFlawAffectsModel(flaw: Ref<ZodFlawType>) {
 
   function refreshAffects() {
     return getFlaw(flaw.value.uuid).then((response) => {
-      console.log('refreshing affects');
+      console.debug('refreshing affects');
```
Comment (superbuggy) [inline src/composables/useFlawAffectsModel.ts:26]: I think we could probably remove this, too

File: src/stores/UserStore.ts [hunk 13]
```diff
@@ -121,7 +121,7 @@ export const useUserStore = defineStore('UserStore', () => {
   }
 
   async function login(username: string = '', password: string = '') {
-    console.log(osimRuntime.value.backends.osidbAuth);
+    console.debug(osimRuntime.value.backends.osidbAuth);
```
Comment (superbuggy) [inline src/stores/UserStore.ts:124]: I think we could remove this

**Review by superbuggy — APPROVED** [review]

**Pull request merged**